subworkflows/local/loadSampleSheet/main.nf

```diff
@@ -0,0 +1,45 @@
+/***********
+| WORKFLOW |
+***********/
+
+workflow LOAD_SAMPLESHET {
```

Comment:
Typo (here and in places it's called): sampleshet -> samplesheet

Reply:
quite the feat on me not noticing that typo.